doc/usermanual/darkroom/modules/effect/watermark.xml

```diff
@@ -374,7 +374,7 @@
                 $(Xmp.dc.description)
               </entry>
               <entry>
-                The description of the image
+                The description of the image.  Note: some cameras write user-specified text at shoot time to exif field 0x9286 a.k.a. 37510 or "User Comment" ("Exif.Photo.UserComment" as per exif2).  At image import time, darktable copies content from that field here.
```

Comment:
I wonder if this additional explanation is too technical. Why not just:
_The description of the image. Note: some cameras allow to write write user-specified text to the picture's metadata. When images are imported to darktabe, the image description is read from the exif data if present._

Reply:
Personally I like the technical explanation.

In addition I think that at least the "Exif.Photo.UserComment" reference is great so that those who like to fiddle with their exif data know what goes where.

(just my 2c)

Reply:
# Upside

> I think that at least the "Exif.Photo.UserComment" reference is great so that those who like to fiddle with their exif data know what goes where.

Yes! The rationale for including the technical name of exif field is to provide attract search engines when they are asked requests like "darktable exif usercomment".
Clearly, people interested in knowing where this field is available in darktable will be glad to find this information.

# Any downside ?

> I wonder if this additional explanation is too technical.

The fact is, Darktable is not for everyone, it already selects an audience interested in more than point-shoot-share.  
What would be an actual downside? Those using darktable and reading the manual, will they be driven off by the mention of an Exif field name?

# Precedent: how Apple does it

> Why not just: *(long text which avoids to explicitly mention the exif field name)*

This reminds me about Apple allowing SSH access to Mac OS X machine. Apple is very wary about feeling "easy to use" and "not too technical".

Very first release of Mac OS X did not dare mention "SSH" in preference panel. Instead there was a circumlocution. That made things difficult for people who actually needed SSH, because they did not recognize it was indeed SSH.

Then Apple figured out it was okay to [call a spade a spade](https://en.wikipedia.org/wiki/Call_a_spade_a_spade). From that time, SSH was always explicitly mentioned in "Preferences" panel, in firewall settings, in knowledge base.

Mac OS X preference panel image (courtesy [stocksy.co.uk - Mac - SSH on Mac OS X](http://www.stocksy.co.uk/articles/Mac/ssh_on_mac_os_x/ "stocksy.co.uk - Mac - SSH on Mac OS X")):
![Mac OS X preference panel](http://www.stocksy.co.uk/images/ssh/sharing.png)

Mentioned in [macos - Configure OSX Firewall to Allow SSH Server? - Super User](https://superuser.com/questions/265856/configure-osx-firewall-to-allow-ssh-server "macos - Configure OSX Firewall to Allow SSH Server? - Super User"):
![Mac OS X firewall settings](https://i.stack.imgur.com/gHoXH.png)

[From Apple knowledge base article: macOS Sierra: Allow a remote computer to access your Mac](https://support.apple.com/kb/PH25252):

> If you allow remote login, you can use Secure Shell (SSH) to log in to your Mac from another computer.

## Conclusion

I believe it's okay to mention "Exif.Photo.UserComment" in manual, as it will not drive off users, yet help people who needs this information in the manual.

Thank you for your attention.

Reply:
I think mentioning ``Exif.Photo.UserComment`` will provide a useful information for some users and is well justified. However, ``exif field 0x9286 a.k.a. 37510 or "User Comment" ("Exif.Photo.UserComment" as per exif2)``, is a kind of wording that I would expect to find in a developer's manual rather than in a user manual.

Reply:
I agree with @rabauke, mentioning the hex number of the EXIF field gives no added information to users and looks more like a garbled file than actually intended content.